flink-connectors/flink-connector-hive/pom.xml

```diff
@@ -862,10 +866,6 @@ under the License.
 					<groupId>org.slf4j</groupId>
 					<artifactId>slf4j-log4j12</artifactId>
 				</exclusion>
-				<exclusion>
-					<groupId>org.pentaho</groupId>
-					<artifactId>pentaho-aggdesigner-algorithm</artifactId>
-				</exclusion>
```

Comment:
While it was excluded here, there is no Pentaho included in `hive-webhcat-java-client` according to the dependency tree